FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,71 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-        
-    }
+		client.get(from: url) {
+			[weak self, completion] (result: Result) in
+			guard self != nil else { return }
+			switch result {
+			case let .success((data, response)):
+				completion(FeedLoader.Result {
+					try mapToFeedImage((data, response))
+				})
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+private func mapToFeedImage(_ result: (Data, HTTPURLResponse)) throws -> [FeedImage] {
+	let (data, response) = result
```

Comment:
This method could receive the `data` and `response` as separate params, so it wouldn't need the tuple destructuring.

```suggestion
private func mapToFeedImage(_ data: Data, _ response: HTTPURLResponse) throws -> [FeedImage] {
```